features/270-next-skycli/commands.md

```diff
@@ -376,10 +376,18 @@ aws_access_secret       2019-01-31T15:00:00+08:00
 
 ### skycli secret create
 
-`skycli secret create [SECRET_NAME] [SECRET_VALUE]`
+`skycli secret create [SECRET_NAME] [SECRET_VALUE] --type=[SECRET_TYPE] --file==[FILE_NAME]`
 
 Create new secret in app
 
+#### Flags
+
+- `--type` Secret type, used to facilitate programmatic handling of secret data. Available types: `opaque`, `dockerconfigjson`. Default is `opaque`.
+    - `opaque`: Secret used in environment variable.
+    - `dockerconfigjson`: Docker config used as image pull secret.
+
+- `--file` Secret value from file.
```

Comment:
Is `--file` specific to `--type=dockerconfigjson` or is it applicable to `--type=opaque` too, just like k8s?

Reply:
I think file should be independent to type, both type can use file.